lldb/source/Plugins/Platform/Linux/PlatformLinux.cpp

```diff
@@ -34,6 +34,20 @@
 #define MAP_PRIVATE 2
 #define MAP_ANON 0x20
 
+// For other platforms that use platform linux
+#ifndef SIGILL
+#define SIGILL 4
+#endif
+#ifndef SIGBUS
+#define SIGBUS 7
+#endif
+#ifndef SIGFPE
+#define SIGFPE 8
+#endif
+#ifndef SIGSEGV
+#define SIGSEGV 11
```

Comment:
You mentioned offline that we define these somewhere else as well? Would be good to have a cleanup to unify them into a single header I think.